opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -518,6 +518,28 @@ message DoubleHistogramDataPoint {
   // (Optional) List of exemplars collected from
   // measurements that were used to form the data point
   repeated DoubleExemplar exemplars = 8;
+
+  // Note: the definition used to for min and max fields below is
+  // based on temporality in a way meant to match the Prometheus
+  // Summary data type used with quantiles 0 and 1, respectively, when
+  // paired with cumulative temporality.  This uses a fixed width that
+  // SHOULD be up to 5-10 minutes or for as long as data is available.
+  // https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#summary
+  
+  // min is the minimum value over a recent window of time.
+  //
+  // The interpretation is based on Metric's temporality.
+  // - delta temporality: the min value MUST be computed over over [start_time, end_time]
+  // - cumulative temporality: the min value SHOULD be computed over [now - WIDTH, end_time]
+  //   for WIDTH in the 5-10 minute range.
+  double min = 13;
```

Comment:
I kind of view this as an "exemplar with default 'min' or 'max' labels", given we can represent exemplars outside of buckets.

Reply:
Since the spec PR has added these fields to the data model, now I feel very little has to be said in the protocol. PTAL.